apps/src/code-studio/signup.js

```diff
@@ -57,8 +59,11 @@ window.SignupManager = function (options) {
     $("#schoolname-block").hide();
 
     // Show correct terms below form.
-    $("#teacher-terms").hide();
     $("#student-terms").fadeIn();
+    $("#teacher-terms").hide();
+
+    // Implicitly accept terms of service for students
+    $("#user_terms_of_service_version").prop('checked', true);
```

Comment:
That is my assumption, given that we aren't prompting students to accept the TOS and PP...